src/Boot/DrupalBoot8.php

```diff
@@ -273,11 +274,6 @@ public function addDrupalModuleDrushCommands(BootstrapManager $manager): void
             $commandInfoAlterers = array_merge($commandInfoAlterers, $commandInfoAlterersInThisModule);
         }
 
-        // Look up the generators from the legacy service instantiator and inject
-        // them into the service manager. The `generate` command will get the
-        // generators from the service manager.
-        $this->serviceManager->injectGenerators($legacyServiceInstantiator->taggedServices('drush.generator.v3'));
-
```

Comment:
We have no attempt at backward compat since generator API changed in Drush 12 anyway.